xpk/Run_MaxText.md

```diff
@@ -39,6 +39,24 @@ curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key --keyr
 sudo apt update && sudo apt-get install google-cloud-sdk-gke-gcloud-auth-plugin
 ```
 
+* Authenticate gcloud installation by running this command and following the prompt
+```
+gcloud auth login
+```
+* Run this command to configure docker to use docker-credential-gcloud for GCR registries:
+```
+gcloud auth configure-docker
+```
+
+Test the installation by running
```

Comment:
nit: adding `*` bullets for ln 51, 55, and 59 so that each section has a bullet point. Thanks!

Reply:
Since that was written as test rather than necessary instructions I omitted bullets, but happy to shortly

Reply:
Done, thanks.